app/Http/Controllers/Admin/ProjectController.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\ProjectRequest;
+use App\Project;
+use Illuminate\Http\Response;
+use Illuminate\Support\Str;
+
+/**
+ * Class ProjectController
+ *
+ * @package App\Http\Controllers\Admin
+ */
+class ProjectController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @author Ali, Muamar
+     *
+     * @return Response
+     */
+    public function index()
+    {
+        return view('admin.pages.projects.index', ['projects' => Project::all()]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @author Ali, Muamar
+     *
+     * @return Response
+     */
+    public function create()
+    {
+        return view('admin.pages.projects.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  ProjectRequest $projectRequest
+     *
+     * @author Ali, Muamar
+     *
+     * @return Response
+     */
+    public function store(ProjectRequest $projectRequest)
```

Comment:
That is the default controller method names because in Laravel they have a naming convention in controller methods names.
"For the controller methods the conventional names are index, show, create, store, edit, update and delete. And snakeCase for custom methods."

Reply:
Liord it is shit it don't even respect psr but ok noted thanks.